stable/artifactory-ha/files/nginx-artifactory-conf.yaml

```diff
@@ -1,5 +1,12 @@
 {{- if .Values.nginx.https.enabled }}
+{{- if .Values.nginx.https.protocols }}
+ssl_protocols {{ .Values.nginx.https.protocols }};
+{{- else -}}
 ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3;
```

Comment:
From a security perspective TLS 1 and TLS 1.1 protocols are outdated and considered vulnerable to various attacks (such as BEAST, POODLE, and others). We shouldn't include them by default

Reply:
I agree, but didn't want to introduce any more variance. I've made those changes. Resolving.